_posts/2018-01-25-fair.md

```diff
@@ -0,0 +1,65 @@
+datasets are large, multidimensional and complex. Perhaps most importantly it
+is probably unrealistic to suggest that a single metadata standard will handle
+the huge diversity of imaging experiments and datasets. In the best possible
+case, it is likely that there will be families of metadata standards or
+flexible APIs, each tuned and designed for accessing specific types of imaging
+metadata.
+ 
+OME has been working on the image data publication problem for many years.
+Our recent work on the [Image Data Resource (IDR)](https://idr.openmicroscopy.org/) is an example of an added
+value database that integrates imaging data from many biological imaging
+datasets and links gene and drug perturbations with cell phenotypes[^2]. IDR
+focuses on reference image datasets, i.e. those datasets that have high levels
+of biological and molecular annotations and have a strong likelihood of re-use
+by the scientific community.
+ 
+Our work on IDR has been well-received and the resource is growing in size and
+usage.  However, IDR doesn’t address more routine data publication, for
+example the datasets that are not reference images, but are associated with a
+scientific publication in the biological sciences. For example, our lab in
+Dundee has recently published a paper that explores the interaction of a
+single protein Bod1 and the Ndc80 complex, a protein complex that mediates the
+attachment of microtubules to connect to chromosomes during cell division[^3].
+How to publish the imaging data associated with this paper?
+ 
+As you might guess, we’ve used OMERO to publish and link these data.  We’ve
+used our institutional OMERO server, and used an institutional DOI as the
+definitive link to the data. The datasets associated with this paper were
+imported into OMERO as part of the analysis workflows and then were moved into
+a public OMERO group for publication. The data can be browsed, searched,
+viewed and downloaded. We believe we’ve made the datasets “AIR”—Accessible,
+Interoperable and Reusable. Making these datasets truly Findable will take
+more time as we develop routine landing pages and JSON-LD-based metadata for
+these images.
+ 
+In the meantime, we thought it might be useful for the community to see how we
+have achieved this work. With the latest releases of OMERO (5.4 and beyond),
+we have made it fairly easy for images to be managed and published online.
+Documentation describing exactly what we did is available[^4].
+ 
+We hope this work is an important contribution to the movement for making the
+data available online. We believe we’ve made reasonable progress in making
+data AIR and look forward to fully achieving the goals of the FAIR principles.
+
+[^1]: [https://www.force11.org/group/fairgroup/fairprinciples](https://www.force11.org/group/fairgroup/fairprinciples)
+[^2]: [Williams et al, 2017](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5536224/)
+[^3]: [Schleicher et al, 2017](http://dx.doi.org/10.1098/rsob.170099)
```

Comment:
change link to http://rsob.royalsocietypublishing.org/content/7/11/170099

Reply:
As per Josh's comment on https://github.com/openmicroscopy/ome-documentation/pull/1830, I'll leave this as is

Reply:
that's fine